src/SelectPopover/SelectPopover.tsx

```diff
@@ -99,21 +100,26 @@ const SelectPopover: FC<SelectPopoverProps> = (
         setPopoverElement(node);
     }
 
-    const popover = useMemo(() => (
+    const popover = useMemo(
+      () => (
         <>
-            {
-                visible && options.length > 0 && (
-                    <StyledSelectPopover>
-                        {
-                            options.map(({ name, value, key }) => (
-                                <SelectPopoverItem key={key ?? name} onClick={() => handleClick(value)}>{name}</SelectPopoverItem>
-                            ))
-                        }
-                    </StyledSelectPopover>
-                )
-            }
+          {visible && options.length > 0 && (
+            <StyledSelectPopover>
+              {options.map(({ name, value, key, disabled }) => (
+                <SelectPopoverItem
+                  key={key ?? name}
+                  onClick={() => !disabled && handleClick(value)}
+                  disabled={disabled}
```

Comment:
Seeing as disabled is an optional arg, what happens when it is not included? Is it falsy and thus not disabled?

Reply:
Yeah, if it's  not included/undefined, it'll just be interpreted as falsy.